bluepyopt/ephys/locations.py

```diff
@@ -133,7 +133,7 @@ def instantiate(self, sim=None, icell=None):
     def __str__(self):
         """String representation"""
 
-        return '%s[%s]' % (self.seclist_name, self.sec_index)
+        return '%s: %s[%s]' % (self.name, self.seclist_name, self.sec_index)
```

Comment:
Ok, but in that case, shouldn't that name get displayed when the user calls str() on the object?